tests/pytests/unit/modules/test_postgres.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import salt.modules.postgres as postgres
+
+# 'md5' + md5('password' + 'username')
+md5_pw = "md55a231fcdb710d73268c4f44283487ba2"
```

Comment:
if possible we want to avoid module level variables in tests

Reply:
Every other pytest you have that I looked at also has module level variables.

You want the constants to be fixtures instead?

Reply:
yes a fixture. If thats the case those would most likely need to be cleaned up then. I'll document this as best practice for when this comes up again.

Reply:
On closer inspection, I don't actually know how to do this parametrized test where some of the values should be fixtures.

Do you have any idea? Could just call the functions but then they're not functioning as fixtures.

Reply:
Looks like you are right. This is not currently supported (https://github.com/pytest-dev/pytest/issues/349)

ping @s0undt3ch you know of any alternative ways to approach this?

Reply:
Also, here's the documentation PR https://github.com/saltstack/salt/pull/59623

Reply:
I misled @Ch3LL with regards to this no globals issue.
Preferably, no globals should be present in test modules, specially mutable globals, like lists and dicts.
The reason for this is that we've seen these global variables being imported in other test modules, mutated and causing tests to fail when running under the full test suite, but not when running alone.

I belive we could still use fixtures for your scenario, but it would involved in indirect fixtures, and the global variable is a string, so, _acceptable_...